src/core/BUILD

```diff
@@ -4555,6 +4555,7 @@ grpc_cc_library(
         "absl/base:core_headers",
         "absl/container:inlined_vector",
         "absl/functional:bind_front",
+        "absl/functional:overload",
```

Comment:
This shouldn't be needed, as per my comment elsewhere about using our `Match()` utility function instead.